tldw_Server_API/tests/DB_Management/test_media_db_legacy_content_query_imports.py

```
@@ -0,0 +1,84 @@
+    assert not hasattr(media_db_v2, "fetch_keywords_for_media")
+    assert not hasattr(media_db_v2, "fetch_keywords_for_media_batch")
+    assert media_item.fetch_keywords_for_media is legacy_content_queries.fetch_keywords_for_media
+    assert (
+        media_listing.fetch_keywords_for_media_batch
+        is legacy_content_queries.fetch_keywords_for_media_batch
+    )
+
+    monkeypatch.setattr(
+        legacy_content_queries,
+        "fetch_keywords_for_media",
+        lambda media_id, db_instance: ["alpha", "beta"],
```

Ruff (0.15.6)

[warning] Unused lambda argument: `media_id`

(ARG005)